sdk/dslink/src/main/java/org/dsa/iot/dslink/serializer/SerializationManager.java

```diff
@@ -25,6 +27,10 @@
     private final Serializer serializer;
     private ScheduledFuture<?> future;
 
+    private Cipher cipher;
```

Comment:
Yeah, I originally had two ciphers but sloppily merged the code into the serialization manager.  However, I did some homework, and the real expensive item to create is the secret key, so I left that.